subjects/corewar/README.md

```diff
@@ -0,0 +1,366 @@
+
+In this case `10` will say to the VM to look ten bytes forward the current instruction and take the value from there.
+
+> Don't be scared to use the provided Asm and VM to play with those different parameters in order to be sure you understood properly their behavior.
+
+#### Instruction Set
+
+| Name  | Nb Params | Opcode | Nb Cycles | Has Pcode | Has Idx | Params                                                                     |
+| ----- | --------- | ------ | --------- | --------- | ------- | -------------------------------------------------------------------------- |
+| live  | 1         | 1      | 10        | false     | false   | Direct                                                                     |
+| ld    | 2         | 2      | 5         | true      | false   | [Indirect, Direct]<br> Register                                            |
+| st    | 2         | 3      | 5         | true      | false   | Register<br>[Register, Indirect]                                           |
+| add   | 3         | 4      | 10        | true      | false   | Register<br> Register<br> Register                                         |
+| sub   | 3         | 5      | 10        | true      | false   | Register<br> Register<br> Register                                         |
+| and   | 3         | 6      | 6         | true      | false   | [Register, Indirect, Direct]<br> [Register, Indirect, Direct]<br> Register |
+| or    | 3         | 7      | 6         | true      | false   | [Register, Indirect, Direct]<br> [Register, Indirect, Direct]<br> Register |
+| xor   | 3         | 8      | 6         | true      | false   | [Register, Indirect, Direct]<br> [Register, Indirect, Direct]<br> Register |
+| zjmp  | 1         | 9      | 20        | false     | true    | Direct                                                                     |
+| ldi   | 3         | 10     | 25        | true      | true    | [Register, Indirect, Direct]<br> [Register, Direct]<br> Register           |
+| sti   | 3         | 11     | 25        | true      | true    | Register<br> [Register, Indirect, Direct]<br> [Register, Direct]           |
+| fork  | 1         | 12     | 800       | false     | true    | Direct                                                                     |
+| lld   | 2         | 13     | 10        | true      | false   | [Indirect, Direct]<br> Register                                            |
+| lldi  | 3         | 14     | 50        | true      | true    | [Register, Indirect, Direct]<br> [Register, Direct]                        |
+| lfork | 1         | 15     | 1000      | false     | true    | Direct                                                                     |
+| nop   | 1         | 16     | 2         | true      | false   | Register                                                                   |
+
+
```

Comment:
General minor comments for all the below points: whenever there is a "bracket statement" to close the brief description I would either (1) always use capital letters for the first word inside the parenthesis or (2) move the `.` at the end of the brackets and keep lower case for the first word in the parenthesis

Reply:
done to uppercase